thunder/tests/distributed/test_ddp.py

```diff
@@ -1039,12 +1041,12 @@ def __call__(self, test_stub):
         init_method = f"{FILE_SCHEMA}{file_name}"
 
         @wraps(test_stub)
-        def test_fn(executor, devices, dtype, bucket_size_in_mb=0):
```

Comment:
This makes it so that we can only use this argument name in the wrapped function with pytest-fixture.